Source/Core/AudioCommon/PulseAudioStream.h

```diff
@@ -17,7 +17,7 @@ class PulseAudio final : public SoundStream
 {
 #if defined(HAVE_PULSEAUDIO) && HAVE_PULSEAUDIO
 public:
-  PulseAudio();
+  PulseAudio() = default;
```

Comment:
This should still be defaulted within the cpp file, since the class contains non-trivial members.

Reply:
done